docs/basics/beginners-contributing-guide.md

```diff
@@ -0,0 +1,119 @@
+# Beginners Contributing Guide
```

Comment:
This page mirrors a lot of https://github.com/selfhostedshow/wiki/blob/24e0b21ad9801e48db4af280c02d4cf354e93e64/CONTRIBUTING.md#how-to-contribute, so we should probably move the missing content of that in here, and just link to here directly.

Reply:
idk why the files saved that way with the extensions but easy fix, in terms of move content from and to https://github.com/selfhostedshow/wiki/blob/24e0b21ad9801e48db4af280c02d4cf354e93e64/CONTRIBUTING.md#how-to-contribute, page i might need more info on what we want that to look like

Reply:
This page should be specifically how to contribute in terms of making a change to the wiki. The `CONTRIBUTING.md` should be more about what contributions are wanted / high level process for PR / review cycle. Most of that it already has, which is good, but now it's expanded more here.

Reply:
ok that works for me i guess i can take a stab at linking and see if thats what you were thinking.